File: common/src/test/java/dev/number6/generate/ChannelMessagesGenerator.java
package dev.number6.generate;

import dev.number6.message.ChannelMessages;
import uk.org.fyodor.generators.Generator;

import java.time.LocalDate;

public class ChannelMessagesGenerator implements Generator<ChannelMessages> {

    @Override
    public ChannelMessages next() {
        return new ChannelMessages(CommonRDG.string().next(), CommonRDG.list(CommonRDG.string()).next(), LocalDate.now());
    }
}

File: common/src/test/kotlin/dev/number6/generate/ChannelMessagesGenerator.kt
package dev.number6.generate

import dev.number6.message.ChannelMessages
import uk.org.fyodor.generators.Generator
import uk.org.fyodor.generators.RDG
import java.time.LocalDate

class ChannelMessagesGenerator : Generator<ChannelMessages> {
    override fun next(): ChannelMessages {
        return ChannelMessages(RDG.string().next(), RDG.list(RDG.string()).next(), LocalDate.now())
    }
}